File: VB/Models/NorthwindDataProvider.vb
Imports System.Data
Imports System.Data.SqlClient
Imports System.Web.Configuration

Public Class NorthwindDataProvider
	Public Shared Function GetOrders() As Object
		Dim connection As New SqlConnection(WebConfigurationManager.ConnectionStrings("NorthwindConnectionString").ConnectionString)
		Dim selectCommand As New SqlCommand("SELECT [OrderID], [CustomerID], [EmployeeID], [OrderDate], [RequiredDate], [ShippedDate] FROM [Orders]")
		Dim da As New SqlDataAdapter(selectCommand)
		Dim ds As New DataSet()

		selectCommand.Connection = connection
		da.Fill(ds, "Orders")

		Return ds.Tables(0)
	End Function
End Class

File: CS/Models/NorthwindDataProvider.cs
using System.Data;
using System.Data.SqlClient;
using System.Web.Configuration;

public class NorthwindDataProvider {
    public static object GetOrders() {
        SqlConnection connection = new SqlConnection(WebConfigurationManager.ConnectionStrings["NorthwindConnectionString"].ConnectionString);
        SqlCommand selectCommand = new SqlCommand("SELECT [OrderID], [CustomerID], [EmployeeID], [OrderDate], [RequiredDate], [ShippedDate] FROM [Orders]");
        SqlDataAdapter da = new SqlDataAdapter(selectCommand);
        DataSet ds = new DataSet();
        
        selectCommand.Connection = connection;
        da.Fill(ds, "Orders");

        return ds.Tables[0];
    }
}